src/frontend/staff-portal/src/app/config/config.service.ts

```diff
@@ -245,4 +242,21 @@ export class ConfigService {
     return (a: Config<number | string>, b: Config<number | string>) =>
       this.utilsService.sortByKey<Config<number | string>>(a, b, 'name');
   }
+
+  private loadCourthouseData(): void {
+    this.http.get<{ courthouses: CourthouseTeam[] }>('/assets/courthouse-data.json')
+      .pipe(
+        map(response => response.courthouses),
```

Comment:
```suggestion
        first(),
        map(response => response.courthouses),
```
I think we need a first() so that this get request auto unsubscribes.

Reply:
Done